src/main/java/com/dev/cinema/controller/OrderController.java

```diff
@@ -34,17 +35,26 @@ public OrderController(OrderService orderService,
     }
 
     @GetMapping
-    public List<OrderResponseDto> getOrderHistory(@RequestParam Long userId) {
+    public List<OrderResponseDto> getOrderHistory(Authentication authentication) {
+        UserDetails details = (UserDetails) authentication.getPrincipal();
+        String email = details.getUsername();
         return orderService
-                .getOrdersHistory(userService.getById(userId))
+                .getOrdersHistory(userService
+                        .findByEmail(email)
+                        .orElseThrow(() -> new RuntimeException("Incorrect login or password")))
```

Comment:
At this moment we will get user, that 100% is in our database. So we can remove this